### DISCUSSION TITLE
How to check an account supports EVM or not? (without showing error to the user)

### DISCUSSION THREAD

**Discussion by RanGojo** [phantom/docs#95] [Help & Support] [answered]
### Summary

When a dApp tries to connect to Phantom EVM but the user account is not compatible and only supports Solana, he faces with an error popup with this message:
`Unsupported Account, localhost:3000 is try to use an EVM network that this Solana private key account doesn't support.`

For a multichain dApp, I would like to connect only to the Solana and skip connecting to the EVM if account doesn't support. My question is :
Is there any way to check an account supports EVM or not ? (without showing users any error popups).

Thanks

### Steps to Reproduce

This is not a bug. I would like to understand how can I check if an account is compatible with EVM or not without showing error to the user.

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment, marked as answer]
Hey @RanGojo is this in the case that the user has Phantom set to Solana only mode?
